docs/how-to/h-manage-units.md

```diff
@@ -0,0 +1,17 @@
+# How to deploy and manage units
+
+## Basic Usage
+
+To deploy a single unit of Self Signed Certificates Operator using its default configuration.
+```shell
+juju deploy self-signed-certificates --channel edge
+```
+
+## Replication
+
+Both scaling-up and scaling-down operations are performed using `juju scale-application`:
+```shell
+juju scale-application  self-signed-certificates <desired_num_of_units>
```

Comment:
Are we sure this actually works?

Reply:
In line 17, I commented that it only works with 1 unit. This command is also used to scale down so it's reasonable to add.

Reply:
I will write it more explicitly that scale up is  not supported yet.

Reply:
Done. The content is moved to tutorial overview section.